test/NuGet.Clients.Tests/NuGet.VsExtension.Test/VsCredentialProviderImporterTests.cs

```diff
@@ -66,26 +67,26 @@ public void WhenVstsImportNotFound_WhenNotDev14_ThenDoNotInsertBuiltInProvider()
             _mockDte.Setup(x => x.Version).Returns("15.0.123456.00");
             var importer = GetTestableImporter();
 
-            var result = importer.GetProvider();
+            var results = importer.GetProviders();
 
-            Assert.Null(result);
+            Assert.DoesNotContain(_visualStudioAccountProvider, results);
         }
 
         [Fact]
         public void WhenVstsImportFound_ThenDoNotInsertBuiltInProvider()
-            {
+        {
             _mockDte.Setup(x => x.Version).Returns("14.0.247200.00");
             var importer = GetTestableImporter();
             var testableProvider = new TeamSystem.NuGetCredentialProvider.VisualStudioAccountProvider();
-            importer.ImportedProvider = testableProvider;
+            importer.ImportedProviders = new List<IVsCredentialProvider> { testableProvider };
 
-            var result = importer.GetProvider();
+            var results = importer.GetProviders();
 
-            Assert.IsType<VsCredentialProviderAdapter>(result);
+            Assert.DoesNotContain(_visualStudioAccountProvider, results);
```

Comment:
Does this test still pass?  The type name of the imported provider won't match Microsoft.VisualStudio.TeamFoundation.NuGetCredentialProvider.VisualStudioAccountProvider.